net/chord/chord.go

```diff
@@ -97,6 +107,15 @@ type localVnode struct {
 	OnNewSuccessor func()
 }
 
+// localVnodeData : Data of localVnode for json.Marshal in API
+type localVnodeData struct {
+	Vnode
+	Successors  []*Vnode
+	Finger      []*Vnode
+	Predecessor *Vnode
+	Last_finger int
```

Comment:
Should we use LastFinger here?

Reply:
It has not debug requirement at current. But just for show more information with very low costing.